package.json

```diff
@@ -44,20 +44,19 @@
     "ember-cli-babel": "^6.8.2",
     "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-version-checker": "^1.2.0",
-    "ember-getowner-polyfill": "^1.2.3",
+    "ember-getowner-polyfill": "^2.2.0",
     "hammerjs": "^2.0.8",
     "rsvp": "^3.1.0"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
     "ember-cli": "~2.16.0-beta.1",
-    "ember-cli-changelog": "^0.3.4",
```

Comment:
We shouldn't remove this, I think we still use it

Reply:
thats actually what was causing the babel 5.x error - hence the move to optional deps... that addon is very old - and its polluting which babel is used, b/c its dependencies has babel 5.x set...

have u seen this issue anywhere else?

same thing with bower, ember-try dies with exit code 1 on last ember-default test w/o it (although we should not be using bower either)

Reply:
We should update ember-cli-changelog. I will try to remember to do that.